src/imbalances_script.py

```diff
@@ -235,14 +236,18 @@ def update_weth_imbalance(
         address: str,
     ) -> None:
         """Update the WETH imbalance in imbalances."""
-        weth_inflow = imbalances.get(WETH_TOKEN_ADDRESS, 0)
-        weth_outflow = 0
-        weth_withdrawals = 0
-        for event in events["WithdrawalWETH"]:
+        weth_transfer_imbalance = imbalances.get(WETH_TOKEN_ADDRESS, 0)
+        weth_withdrawals, weth_deposits = 0, 0
```

Comment:
Why are we using 2 variables here? Couldn't we simply have a `weth_imbalance` variable and add or subtract things to that in case we have a deposit or withdrawal?